apps/README.md

```diff
@@ -0,0 +1,26 @@
+`FluentUI Tester` is the test app that we use to test our FluentUI components during development.
+
+## Launch `FluentUI Tester` app on iOS
+
+Prereq: FluentUI Tester on iOS can only run on a Mac.
+
+1. Make sure you have followed the Getting Started instructions [here](../../README.md) to install packages and build the entire Fluent UI React Native repository. E.g. from the root of the repo:
+```
+    yarn && yarn build
+```
+
+2. Then go into `apps/ios/src` folder and run pod install:
```

Comment:
Should probably explain why we need this (just like how you explained why we need to run yarn and yarn build in #1 above). For instance mentioning that pod install pulls in the project-level Cocoapod dependencies defined in the podfile and generates a valid `xcworkspace` one can use

Reply:
Thanks, added your explanation in new commit.